spec/feature/authenticated_user_spec.rb

```diff
@@ -192,7 +193,7 @@ def click_add_to_cart_link(category)
 
   def valid_user_logs_in
     click_link_or_button "Log In"
-    fill_in 'session_email', with: "rich.shea@gmail.com"
+    fill_in 'session_email', with: "rich@gmail.com"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.